.eslintrc.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
```

Comment:
Does it not support JSON or ESM?

Reply:
If CJS is the only option, can we use a `.cjs` extension?

Reply:
I switched to JSON with comments.

Reply:
I see both files now?

Reply:
Oops. Forgot to commit. 😅 Should be fixed now.